plugins/language_rivet.lua

```diff
@@ -30,9 +30,11 @@ syntax.add {
         },
         {pattern = "[A-Z][%w_]*", type = "keyword2"}, -- types
         {pattern = "%@%s?[%a_][%w_]*", type = "literal"}, -- builtin func/var
+        -- `defer` modes
+        {pattern = "defer%s?%(%s?()[%a_][%w_]*()%s?%)", type = {"keyword", "comment"} },
```

Comment:
There are 3 "groups", so there should be 3 `type`s.

Reply:
Oops, that way the word in parentheses `(MODE)` was highlighted